src/actions/authActions.jsx

```diff
@@ -1,8 +1,45 @@
+const splitFullName = (fullName, selection) => {
```

Comment:
you should never do it like this, please just store the full name in the db or let the user pick first or last name. the reason is that you cannot assume anything about anybody's names. some people have first names that are multiple words. some people don't have last names (yes, this is actually a thing)

https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/

Reply:
read the last pargraph especially ;)

Reply:
that's a perfect article, btw I faced this problem and I assumed that except dealing with (Japanese, Chinese, or Korean ), in general, people's names are tow words: first and last name (general case), and the special case is they may have multiple names, to solve that problem I choose the very first word as a first name and rest as the last name.
can you please give me your advice to solve the issue of the 1st and last name, because this problem happens only with auth with Facebook or Google.